web/ui/static/js/graph.js

```diff
@@ -518,7 +519,6 @@ Prometheus.Graph.prototype.transformData = function(json) {
       color: palette.color()
     };
   });
-  Rickshaw.Series.zeroFill(data);
```

Comment:
That test doesn't indicate one way or the other, as there's no gap where no series exists. The code indicates it's not sufficient.